cmd/shell.go

```diff
@@ -207,6 +207,15 @@ func StatsAndLogging(statConf StatsdConfig, logConf SyslogConfig) (metrics.Statt
 	return stats, logger
 }
 
+func ReportDbConnCount(dbMap *gorp.DbMap, statter metrics.Statter, prefix string) {
```

Comment:
Since we can't automatically run this as part of `StatsAndLogging`, it makes more sense to put it in `sa/database.go`